plugins/mimecast/komand_mimecast/util/api.py

```diff
@@ -115,6 +116,9 @@ def get_siem_logs(self, next_page_token: str) -> Union[List[Dict[str, Any]], Dic
                 self._handle_siem_logs_error_response(response, status_code)
         except json.JSONDecodeError as json_error:
             error_info = f'response content="{response.content}", response headers="{response.headers}"'
+            self.logger.info(
```

Comment:
I think we need this logged on line 216, and the extra couple of exceptions within `_handle_zip_file` as it never hits this exception? those ones we hit and drop out because w'eve got the bad data from mimecast.